app/Services/UserService.php

```diff
@@ -27,25 +27,25 @@ public function __construct(IUserRepository $userRepository)
      * 
      * NOTE: Fix response (to follow)
      */
-    public function verifyEmail(int $userId): string
+    public function verifyEmail(int $userId, bool $markAsVerified = false): string
     {
         $user = $this->userRepository->findById($userId);
 
         if (!$user) {
             return 'User not found.';
         }
+        
+        if ($this->userRepository->isTokenExpired()) {
+            return 'Your email verification is already expired. Please resend a new email verification using the app!';
```

Comment:
nitpick

"Your email verification has expired. Please resend a new email verification using the app!"